Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -481,6 +489,7 @@ def __init__(self,
         synapse_analytics_resource_id='https://dev.azuresynapse.azure.cn',
         portal='https://portal.azure.cn',
         keyvault='https://vault.azure.cn',
+        exchange_online='https://partner.outlook.cn'
```

Comment:
[endpoint ref](https://learn.microsoft.com/en-us/microsoft-365/enterprise/urls-and-ip-address-ranges-21vianet?view=o365-worldwide&viewFallbackFrom=o365-worldwid#:~:text=No-,partner.outlook.cn,-40.73.132.0/24%2C%2040.73.164.128)